### DISCUSSION TITLE
🐛 [BUG]面积图 / 图形样式/ areaStyle / lineDash配置不生效

### DISCUSSION THREAD

**Discussion by shuaishuai222** [antvis/G2Plot#3579] [Q&A] [answered]
### 面积图 / 图形样式/ areaStyle /  lineDash配置不生效

[demo地址](https://charts.ant.design/zh/examples/area/basic#basic-gradients
)

**代码**

import React, { useState, useEffect } from 'react';
import ReactDOM from 'react-dom';
import { Area } from '@ant-design/plots';

const DemoArea = () => {
  const [data, setData] = useState([]);

  useEffect(() => {
    asyncFetch();
  }, []);

  const asyncFetch = () => {
    fetch('https://gw.alipayobjects.com/os/bmw-prod/1d565782-dde4-4bb6-8946-ea6a38ccf184.json')
      .then((response) => response.json())
      .then((json) => setData(json))
      .catch((error) => {
        console.log('fetch data failed', error);
      });
  };
  const config = {
    data,
    xField: 'Date',
    yField: 'scales',
    xAxis: {
      range: [0, 1],
      tickCount: 5,
    },
    areaStyle: () => {
      return {
        fill: 'l(270) 0:#ffffff 0.5:#7ec2f3 1:#1890ff',
        lineDash: [10, 1]
      };
    },
  };

  return <Area {...config} />;
};

ReactDOM.render(<DemoArea />, document.getElementById('container'));

**Comment by visiky** [comment]
> lineDash配置不上

G2 4.0 版本吗？可否提供一个 codesandbox 复现链接

**Comment by shuaishuai222** [comment]
> > lineDash配置不上
> 
> G2 4.0 版本吗？可否提供一个 codesandbox 复现链接

https://charts.ant.design/zh/examples/area/basic#basic-gradients

1.4.2

**Comment by shuaishuai222** [comment]
@visiky

**Comment by hustcc** [comment, marked as answer]
设置 line.style.lineDash 即可。

```ts
line: {
  style: {
    lineDash: [4, ],
  }
}
```

![image](https://github.com/antvis/G2Plot/assets/7856674/accf32c0-ca14-43bb-b15a-b1fae938d08e)
